Sources/Rendering/Core/Mapper/index.js

```diff
@@ -136,13 +136,14 @@ function vtkMapper(publicAPI, model) {
   };
 
   publicAPI.mapScalars = (input, alpha) => {
-    const scalars = publicAPI.getAbstractScalars(
+    const { scalars, cellFlag } = publicAPI.getAbstractScalars(
       input,
       model.scalarMode,
       model.arrayAccessMode,
       model.arrayId,
       model.colorByArrayName
-    ).scalars;
+    );
+    model.areScalarsMappedFromCells = cellFlag;
```

Comment:
Instead of storing this as a value, (which is technically only valid after mapScalars is called) and creating a getter. Isn't it easier to simply create a getter without having to store the value as a member variable).

Reply:
I agree this is not very clean but `areScalarsMappedFromCells` describes how `colorCoordinates` should be interpreted
So I set it in `mapScalars` because this is where `colorCoordinates` is set
Do you have any other suggestion?

Reply:
I looked at the code, and... I agree with you. Let's leave it as is.